lib/grape/endpoint.rb

```diff
@@ -210,47 +216,49 @@ def equals?(e)
     protected
 
     def run(env)
-      @env = env
-      @header = {}
+      ActiveSupport::Notifications.instrument('run.grape', endpoint: self, env: env) do
+        @env = env
```

Comment:
@dblock I guess I'm confused as to how this helps. The implication here is that there could also be a `run` without instrumentation. What value does this provide?